_data-prepper/common-use-cases/metrics-logs.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+title: Deriving metrics from logs
+parent: Common use cases
+nav_order: 15
+---
+
+# Deriving metrics from logs
+
+You can use Data Prepper to derive metrics from logs. 
+
+The following example pipeline receives incoming logs using the [`http` source plugin]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/http-source) and the [`grok` processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/grok/). It then uses the [`aggregate` processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/aggregate/) to extract the metric bytes aggregated during a 30-second window and derives histograms from the results.
+
+This pipeline writes data to two different OpenSearch indexes:
```

Comment:
@dlvenable Please review the text added at lines 14--27 and let me know if these additions address your feedback and are accurate. Thank you.